### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Krimat**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,51 @@
...
+            id='Test when next rate in between 0 and +0.05'
+        ),
+        pytest.param(
+            1.0,
+            0.97,
+            "Do nothing",
+            id='Test when next rate in between -0.05 and 0'
+        )
+    ],
+)
```
Comment (MasterpieceElbow) [inline tests/test_main.py:42]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by MasterpieceElbow — APPROVED** [review]
